cmd/controller/app/server.go

```diff
@@ -65,9 +65,11 @@ func NewControllerManagerCommand() *cobra.Command {
 		Use:   "controller-manager",
 		Short: `KubeSphere DevOps controller manager`,
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			if errs := s.Validate(); len(errs) == 0 {
-				err = Run(s, signals.SetupSignalHandler())
+			if errs := s.Validate(); len(errs) != 0 {
+				return utilerrors.NewAggregate(errs)
 			}
+
+			err = Run(s, signals.SetupSignalHandler())
```

Comment:
IMO, the code below could simplify `return err = Run(s, signals.SetupSignalHandler())`.

```go
err = Run(s, signals.SetupSignalHandler())
return
```

Reply:
I didn't see the big differences.

Reply:
Clear and consistent are big differences.